nirc_ehr/resources/queries/ehr/protocol.js

```diff
@@ -0,0 +1,35 @@
+
+require("ehr/triggers").initScript(this);
+
+EHR.Server.TriggerManager.registerHandlerForQuery(EHR.Server.TriggerManager.Events.BEFORE_INSERT, 'ehr', 'protocol', function (helper, scriptErrors, row, oldRow) {
+
+    if (row.description) {
+        plainTextParts = row.description.split("\\loch\\af0")
+        let plainText = "";
+        for(let i = 0; i < plainTextParts.length; i++)
+        {
+            if (i > 0)
```

Comment:
Are you intentionally skipping index 0 here?

Reply:
Yes the first part is all RTF formatting